examples/pytorch/BertNewsClassification/conda.yaml

```diff
@@ -14,5 +14,5 @@ dependencies:
   - sklearn
   - cloudpickle==1.6.0
   - boto3
-  - transformers<4.0.0
+  - transformers
```

Comment:
@harupy Does this need to be >= 4.0.0?

Reply:
Yeah, since transforms > 4.0.0 returns a tensor and the ^code does not unpack it, shouldn't it be >= 4.0.0
`output = self.bert_model(input_ids=input_ids, attention_mask=attention_mask)`

Reply:
Thanks for the comments! Fixe it.